package.json

```diff
@@ -52,7 +52,7 @@
     "typescript": "^3.9.5"
   },
   "lint-staged": {
-    "*.src.*{ts,js}": [
+    ".*src.*{ts,js}": [
```

Comment:
How is this different from `*.`?
Will we have `test-src` too? Or something else?
What if we'd have a path `test/mysrcfolder/noRun.js`? It should match.

Reply:
`*.` is a wrong expression (it doesn't work).

There is a task to enable linting in `tests` in the next sprint (I added it last week).